source/auth_n.rst

```diff
@@ -5101,15 +5049,6 @@ If authentication is successful, you will get back the Account:
       .. literalinclude:: code/vbnet/authentication/mfa_auth_account_resp.vb
           :language: vbnet
 
-.. only:: java
-
-  .. todo::
-
-    (java.todo)
-
-    .. literalinclude:: code/java/authentication/mfa_auth_account_resp.java
-        :language: java
-
```

Comment:
Sorry but I am not seeing what @jakubswiatczak mentioned... Has it already been fixed?